DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -15,18 +15,35 @@
 L1TStage2uGT::L1TStage2uGT(const edm::ParameterSet& params):
    l1tStage2uGtSource_(consumes<GlobalAlgBlkBxCollection>(params.getParameter<edm::InputTag>("l1tStage2uGtSource"))),
    monitorDir_(params.getUntrackedParameter<std::string> ("monitorDir", "")),
-   verbose_(params.getUntrackedParameter<bool>("verbose", false))
+   verbose_(params.getUntrackedParameter<bool>("verbose", false)),
+   algoBitFirstBxInTrain_(-1),
+   algoBitLastBxInTrain_(-1),
+   algoNameFirstBxInTrain_(params.getUntrackedParameter<std::string>("firstBXInTrainAlgo","")),
+   algoNameLastBxInTrain_(params.getUntrackedParameter<std::string>("lastBXInTrainAlgo",""))
 {
-   // empty
+   // Get the trigger bits corresponding to the trigger names from the event setup.
```

Comment:
If gtUtils is constructed in the initializer list these five lines can be removed.